spec/integration/kibana/tags.spec.yml

```diff
@@ -0,0 +1,51 @@
+##
+## Describes the specification for a routing rules yml file
+##
+spec:
+  # Everything under here follows JSON schema (https://json-schema.org/), written as YAML for readability
+  type: array
+  description: Tags automatically added to saved object assets.
+  items:
+     type: object
+     additionalProperties: false
+     properties:
+       text:
+         description: Tag name.
+         type: string
+       asset_types:
+         description: Asset types where this tag is going to be added.
+         type: array
+         items:
+           type: string
+           enum: # added the same types that are allowed under kibana folder
+             - dashboard
+             - visualization
+             - search
+             - map
+             - lens
+             - index_pattern
+             - security_rule
+             - csp_rule_template
+             - ml_module
+             - tag
+             - osquery_pack_asset
+             - osquery_saved_query
```

Comment:
These are the asset types that are allowed to be under `kibana` folder:
https://github.com/elastic/package-spec/blob/7990807d58465a669fac5ce95f8857fdbc232128/spec/integration/kibana/spec.yml

Reply:
Just removed `tag` from the list. I think tags would not be assigned other tags.